content/en/website-usability-testing.html

```diff
@@ -7,29 +7,30 @@
 <section class="page--outer-container-padding">
     <div class="row">
         <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+            
             <h2 class="section--title"> We need your help</h2>
```

Comment:
The original heading is causing an accessibility contrast issue. If we remove the class, it gets rid of the issue and is consistent with the other <h2> in the project.

<img width="1751" alt="Screen Shot 2021-08-03 at 10 36 46 AM" src="https://user-images.githubusercontent.com/33768337/128034523-4cd9a9a0-e88b-4ddb-9822-55c6658f39cf.png">